wiki/Tournaments/Donut_Cup/1/en.md

```diff
@@ -332,13 +343,32 @@ The Donut Cup is run by various osu! community members from all over the world.
 
 ## Match results
 
+### Grand Finals[^stage-note-2]
+
+Detailed statistics for this round in [the following spreadsheet](https://docs.google.com/spreadsheets/d/1q6CsZ5b3WHUTXUXfCpVdsJxgTDvz_Q2hRPdDbswgJFQ/edit).
+
+Friday, 12 July 2024:
+
+| Bracket | Team 1 |  |  | Team 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Lower | No title | 1 | **7** | **crying right now** | [#1](https://osu.ppy.sh/community/matches/114644017) |
+
+Sunday, 14 July 2024, Grand Final:
+
+| Bracket | Team 1 |  |  | Team 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Grand Final | YOSHIMOBIL | 6 | **7** | **crying right now** | [#1](https://osu.ppy.sh/community/matches/114669424) |
+| Grand Final (Bracket reset) | **crying right now** | **7** | 3 | YOSHIMOBIL | [#1](https://osu.ppy.sh/community/matches/114670678) |
```

Comment:
table cells are generally sentence-cased, so "bracket" shouldn't be capitalised because it's not a proper noun here

```suggestion
| Grand Final (bracket reset) | **crying right now** | **7** | 3 | YOSHIMOBIL | [#1](https://osu.ppy.sh/community/matches/114670678) |
```